client/templates/nav/nav.js

```diff
@@ -40,15 +40,11 @@ Template.notificationsDropdown.onRendered(function() {
 
 Template.notificationsDropdown.helpers({
   hasNotifications: function(){
-    if(Meteor.user() && Meteor.user().notificationsCount() > 0){
-      return true;
-    }
-
-    return false;
+    return Meteor.user() && Meteor.user().notificationsCount() > 0;
```

Comment:
For reference:

    notificationsCount: function(){
      var notifications = Notifications.find({ownerId: this._id, isRead: false});
      return notifications ? notifications.count() : 0;
    }